source/Pages/compute_on_spider.rst

```diff
@@ -221,17 +225,26 @@ Job types
   * For example, by specifying ``-c 2`` you request 2 cores and 160 GB scratch disk
   * When you target specifically our fat nodes with 12TB available scratch, the provided scratch disk per requested core is 200 GB
 
+GPU jobs
+========
+* For jobs that require GPU resources a specific partition is available (see :ref:`partitions <partitions>` for all the different partitions).
+
+
+.. _partitions:
+
 ================
 Slurm partitions
 ================
 
-We have configured four partitions on :abbr:`Spider (Symbiotic Platform(s) for Interoperable Data
+We have configured four CPU and two GPU partitions on :abbr:`Spider (Symbiotic Platform(s) for Interoperable Data
 Extraction and Redistribution)` as shown in the table above:
 
   * If no partition is specified, the jobs will be scheduled on the normal partition  which has a maximum walltime of 120 hours and can run on any worker nodes.
   * Infinite partition jobs have a maximum walltime of 720 hours. Please note that you should run on this partition at your own risk. Jobs running on this partition can be killed without warning for system maintenances and we will not be responsible for data loss or loss of compute hours.
   * Short partition is meant for testing jobs. It allows for 2 jobs per user with 8 cores max per job and 12 hours max walltime.
   * Interactive partition is meant for testing jobs and has 12 hours maximum walltime.
+  * GPU V100 contains Nvidia V100 cards
+  * GPU A100 contains Nvidia A100 cards
```

Comment:
It may be useful to provide a bit more information on the GPU cards (e.g., number of GPUs and how much memory per card). I would suggest referring this back to the descriptions of the different nodes that we have in Spider - which I hope we have documented. Note that scontrol show node, will not show this type of GPU information (at least not atm) so we should have this somewhere.